### PR TITLE
Meghana reddy step 20

### PR DESCRIPTION
**Pull request by Meghana-Reddy2**
DESCRIPTION

Step 20: Be able to label your tasks
Let's make it possible to attach multiple labels to tasks
Let's make it searchable by the label we added.

Screenshot 2024-03-12 at 18 12 01
Screenshot 2024-03-12 at 18 12 12
Screenshot 2024-03-12 at 18 12 23
Screenshot 2024-03-12 at 18 12 32
Screenshot 2024-03-12 at 18 13 11

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+myapp/bin/webpack
+myapp/bin/webpack-dev-server
```
Comment (yykamei) [inline .gitignore]: Why do you want to exclude these commands from Git?

File: myapp/spec/factories/labels.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :label do
+    name { "MyString" }
+    association :user
```
Comment (yykamei) [inline myapp/spec/factories/labels.rb]: You don't have to explicitly use `association`. Instead, you can just `user` like this:

```suggestion
    user
```

This page might be helpful.

https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#associations

**Review by yykamei — APPROVED** [review]
OK, thank you for learning Ruby on Rails 🛤️

File: .gitignore (hunk 1 above)
Comment (Meghana-Reddy2) [reply .gitignore]: I don't want to display the bin files.

File: myapp/spec/factories/labels.rb (hunk 2 above)
Comment (Meghana-Reddy2) [reply myapp/spec/factories/labels.rb]: Noted. Thank you for the material 🙏

File: .gitignore (hunk 1 above)
Comment (yykamei) [reply .gitignore]: Where are these files displayed? Why is `myapp/bin/rails` allowed to be displayed somewhere while `webpack*` are not allowed?

Comment (Meghana-Reddy2) [reply .gitignore]: Running the server again updated my webpacker version so I think I should include it to be shown. I thought as it was already in my bin files I could just ignore it from displaying again. 👀  
I added the files here [072370d](https://github.com/Fablic/training/pull/1621/commits/072370df243157c2a78f77ce4500e62c802baf58)

Comment (yykamei) [reply .gitignore]: OK. No problem

**Pull request merged**